src/edge/tests/test_views.py

```diff
@@ -404,6 +404,40 @@ def test_add_annotation_on_reverse_strand(self):
             "feature_base_last": 8,
         }])
 
+    def test_annotate_multiple_chunks_with_qualifiers(self):
+        data = dict(bases=((2, 4), (6, 9)),
+                    name='proC', type='promoter', strand=1, qualifiers=dict(gene="PROC"))
+        res = self.client.post(self.uri + 'annotate-chunks/', data=json.dumps(data),
```

Comment:
style thing, but if we named the urls we could get the urls with `reverse` and not have to chase any changes to where the api is laid out in the future

Reply:
will do.